### PR TITLE
Remove obsoleted code for macOS Toggl

### LINKED ISSUE #3110: Delete obsoleted code on macOS Toggl
### 📒 Description
We've implemented completely new design & logic for the TimeEntry Editor and major parts of TimeEntryList. It's perfect time to remove all of old legacy, and re-organize the structure of the project.

### 💻 Environment
Platform: macOS

### Acceptance Criteria
- [x] Remove [TableViewDiff] (https://github.com/toggl/toggldesktop/tree/master/src/ui/osx/TogglDesktop/Diff) and DeepDiff library, since we don't use NSTableView in TimeEntry List anymore
- [x] Remove `TimeEntryEditViewController`, since we have `EditorViewController.swift`
- [x] Remove MKPopoverColorWell, MKColorWellCustom, which belongs in old `TimeEntryEditViewController`
- [x] Remove bunch of old images assets in `images` folders
- [x] Move all Swift file in `ui` folder to different location: It's hard to keep track when the `ui` folder have objc files and swift in the long list.
- [x] Remove `NSUnstripedTableView`, `NSViewEscapable` since we don't use it anywhere
- [x] Remove test target: kopsikTests
- [x] Convert all pdf icon to Image.assets

### PR DESCRIPTION
**Pull request by NghiaTranUIT**
### 📒 Description
We've recently implemented new feature, but we let too much obsoleted code, which we never use again.

It's not fruitful, and it's time to get rid of it.

### 🕶️ Types of changes
**Improvements** (non-breaking change which adds functionality)

### ⚠️ Warning
Please review https://github.com/toggl/toggldesktop-branding/pull/37, since this PR affect how the codesign works

### 🤯 List of changes
- [x] Remove [TableViewDiff] (https://github.com/toggl/toggldesktop/tree/master/src/ui/osx/TogglDesktop/Diff) and DeepDiff library, since we don't use NSTableView in TimeEntry List anymore
- [x] Remove `TimeEntryEditViewController`, since we have `EditorViewController.swift`
- [x] Remove MKPopoverColorWell, MKColorWellCustom, which belongs in old `TimeEntryEditViewController`
- [x] Remove bunch of old images assets in `images` folders
- [x] Move all Swift file in `ui` folder to different location: It's hard to keep track when the `ui` folder have objc files and swift in the long list.
- [x] Remove `NSUnstripedTableView`, `NSViewEscapable` since we don't use it anywhere
- [x] Remove test target: kopsikTests
- [x] Convert all pdf icon to Image.assets
- [x] Remove TFDatePicker library
- [x] Re-organize the Poco's library and update Search Header to `third_party/openssl/include`, Library Search Path = `third_party/openssl`

### 👫 Relationships
Close #3110 

### 🔎 Review hints
- This PR shouldn't change any logic of the app. All logic should be remained, and able to build from the scratch.

### CODE REVIEW

**Comment by NghiaTranUIT** [comment]
Updated warning note for https://github.com/toggl/toggldesktop-branding/pull/37

**Comment by skel35** [comment]
Cloned the repo and built everything from scratch.
Tried out all the basic functionality like entry start/stop/edit, autocomplete, shortcuts, reminders, autotracker, Pomodoro timer, idle detection.
Works well 👍

File: src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2586,6 +2292,7 @@
 					"$(SRCROOT)",
 					../../../lib/osx/build,
 					"$(SRCROOT)/../../../../third_party/poco/lib/Darwin/x86_64",
+					"$(SRCROOT)/../../../../third_party/openssl/",
```
Comment (NghiaTranUIT) [inline src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj]: Here is the last change, which is related to `openssl`

File: src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2574,10 +2280,10 @@
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
-					"$(SRCROOT)/../../../../third_party/TFDatePicker/**",
 					"$(SRCROOT)/../../../../third_party/plcrashreporter/**",
 					"$(SRCROOT)/../../../lib/include",
 					"$(SRCROOT)/../../../../third_party/Sparkle/Sparkle.framework/**",
+					"$(SRCROOT)/../../../../third_party/openssl/include",
```
Comment (NghiaTranUIT) [inline src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj]: Here

File: src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -2629,10 +2335,10 @@
...
```
Comment (NghiaTranUIT) [inline src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj]: Here

File: src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -2641,6 +2347,7 @@
 					"$(SRCROOT)",
 					../../../lib/osx/build,
 					"$(SRCROOT)/../../../../third_party/poco/lib/Darwin/x86_64",
+					"$(SRCROOT)/../../../../third_party/openssl/",
```
Comment (NghiaTranUIT) [inline src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj]: Here

**Comment by IndrekV** [comment]
I think these changes are not connected as I tried validating a bit earlier build that was before these changes and the outcome was the same. I think the issue is that thes libcrypto and libssl dylib libraries were built with older Xcode and do not have some needed parameter coded into the libraries.

**Pull request merged**